bin/node/rpc/src/lib.rs

```diff
@@ -88,6 +88,8 @@ pub struct FullDeps<C, P, SC, B> {
 	pub babe: BabeDeps,
 	/// GRANDPA specific dependencies.
 	pub grandpa: GrandpaDeps<B>,
+	/// Shared statement store reference.
+	pub statement_store: Arc<dyn sp_statement_store::StatementStore>,
```

Comment:
Is `sp_statement_store::StatementStore` really a primitive?

Reply:
It's a trait, not the actual implementation.